### PR TITLE
Add PNG support for transparency

### PR DESCRIPTION
**Pull request by bipinkrish**
Issue for more info : https://github.com/sign-language-processing/pose/issues/82

### CODE REVIEW

File: src/python/pose_format/pose_visualizer.py [hunk 1]
```diff
@@ -128,8 +131,9 @@ def draw(self, background_color: Tuple[int, int, int] = (255, 255, 255), max_fra
 
         """
         # ...
+        background_color += (alpha,)
         int_frames = np.array(np.around(self.pose.body.data.data), dtype="int32")
-        background = np.full((self.pose.header.dimensions.height, self.pose.header.dimensions.width, 3),
+        background = np.full((self.pose.header.dimensions.height, self.pose.header.dimensions.width, 4),
```
Comment (AmitMY) [inline src/python/pose_format/pose_visualizer.py:136]: and then I'd split this into two cases, transparent or not

File: src/python/pose_format/pose_visualizer.py [hunk 2]
```diff
@@ -231,11 +235,45 @@ def save_gif(self, f_name: str, frames: Iterable[np.ndarray]):
         images = [Image.fromarray(self.cv2.cvtColor(frame, self.cv2.COLOR_BGR2RGB)) for frame in frames]
         images[0].save(f_name,
                        format="GIF",
-                       append_images=images,
+                       append_images=images[1:],
```
Comment (AmitMY) [inline src/python/pose_format/pose_visualizer.py:238]: nice catch!

**Review by AmitMY — CHANGES_REQUESTED** [review]
Thanks. Good work! Minor modifications required/should be discussed.

**Comment by bipinkrish** [comment]
All the changes requested are fixed

**Review by AmitMY — APPROVED** [review]
absolutely fantastic.

**Pull request merged**